stock_mts_mto_rule/model/warehouse.py

```diff
@@ -88,15 +88,15 @@ def _get_push_pull_rules_values(
                     })
         return res
 
-    @api.multi
-    def create_routes(self):
-        pull_model = self.env['procurement.rule']
-        res = super(Warehouse, self).create_routes()
-        if self.mto_mts_management:
-            mts_mto_pull_vals = self._get_mts_mto_rule()
+    @api.model
+    def create(self, vals):
+        wh = super(Warehouse, self).create(vals)
+        if wh.mto_mts_management:
```

Comment:
@florian-dacosta Maybe I would have put code here under into separate method (to easy override).